### ISSUE TITLE
client.add(infoHash [in hex]) doesn't appear to work.

### ISSUE THREAD

**Issue by mikeal** [feross/webtorrent#902]
* WebTorrent version:
latest as of today :)
* Node.js version:
6.x cause I'm a responsible developer :P
* Browser name/version (if using WebTorrent in the browser):
Chrome 53

The docs say `client.add()` will work with an infoHash hex but it's not working for me. I literally seed a file in one tab and try to get it in another and it can't load the metadata. If i swap that out for a magnet url it works great.

**Comment by alxhotel** [comment]
Have you set the torrent trackers ?

**Comment by mikeal** [comment]
nope. if `seed()` defaults to a few trackers why doesn't `add()` default to the same trackers when none are specified?

**Comment by alxhotel** [comment]
Check out this comment: [#672 (comment)](https://github.com/feross/webtorrent/issues/672#issuecomment-196614860)

**Comment by mikeal** [comment]
Oh interesting, thanks! :)

**Comment by feross** [comment]
@alxhotel Thanks for linking to the rationale for the decision.

@mikeal Do you think this decision is a bit confusing? Maybe we should keep parallel behavior between `seed()` and `add()` when it comes to omitting trackers?

Happy to reconsider the decision if you think it is confusing for new users.

**Comment by mikeal** [comment]
@feross I think it is confusing for sure, having the same behavior, or possibly giving a warning when omitting any trackers, would clear it up. I get why not having default trackers is a good idea, I just think it would be good to be consistent.

**Comment by DiegoRBaquero** [comment]
@mikeal Did you specify at least 1 custom tracker? If not, then this is a bug, as it should add the default ones.

**Comment by deamme** [comment]
@DiegoRBaquero I think it's a bug because it doesn't add the default ones when a tracker is not specified. It works great with magnetURI though.
